WooCommerce/Classes/ViewRelated/Orders/OrdersRootViewController.swift

```diff
@@ -170,8 +219,86 @@ private extension OrdersRootViewController {
     }
 }
 
-extension OrdersRootViewController: OrdersTabbedViewControllerScrollDelegate {
+extension OrdersRootViewController: OrderListViewControllerDelegate {
+    func orderListViewControllerWillSynchronizeOrders(_ viewController: UIViewController) {
+        // Do nothing here
+    }
+
     func orderListScrollViewDidScroll(_ scrollView: UIScrollView) {
         hiddenScrollView.updateFromScrollViewDidScrollEventForLargeTitleWorkaround(scrollView)
     }
 }
+
+// MARK: - Creators
+
+private extension OrdersRootViewController {
+    /// Create a `UIBarButtonItem` to be used as the search button on the top-left.
+    ///
+    func createSearchBarButtonItem() -> UIBarButtonItem {
+        let button = UIBarButtonItem(image: .searchBarButtonItemImage,
+                                     style: .plain,
+                                     target: self,
+                                     action: #selector(displaySearchOrders))
+        button.accessibilityTraits = .button
+        button.accessibilityLabel = NSLocalizedString("Search orders", comment: "Search Orders")
+        button.accessibilityHint = NSLocalizedString(
+            "Retrieves a list of orders that contain a given keyword.",
+            comment: "VoiceOver accessibility hint, informing the user the button can be used to search orders."
+        )
+        button.accessibilityIdentifier = "order-search-button"
+
+        return button
+    }
+
+    /// Create a `UIBarButtonItem` to be used as a way to create a new simple payments order.
+    ///
+    func createAddSimplePaymentsOrderItem() -> UIBarButtonItem {
+        let button = UIBarButtonItem(image: .plusBarButtonItemImage,
+                                     style: .plain,
+                                     target: self,
+                                     action: #selector(presentSimplePaymentsAmountController))
+        button.accessibilityTraits = .button
+        button.accessibilityLabel = NSLocalizedString("Add simple payments order", comment: "Navigates to a screen to create a simple payments order")
+        button.accessibilityIdentifier = "simple-payments-add-button"
+        return button
+    }
+
+    /// Presents `SimplePaymentsAmountHostingController`.
+    ///
+    @objc private func presentSimplePaymentsAmountController() {
+        let viewModel = SimplePaymentsAmountViewModel(siteID: siteID)
+        viewModel.onOrderCreated = { [weak self] order in
+            guard let self = self else { return }
+
+            self.dismiss(animated: true) {
+                self.navigateToOrderDetail(order)
+            }
+        }
+
+        let viewController = SimplePaymentsAmountHostingController(viewModel: viewModel)
+        let navigationController = WooNavigationController(rootViewController: viewController)
+        present(navigationController, animated: true)
+
+        ServiceLocator.analytics.track(event: WooAnalyticsEvent.SimplePayments.simplePaymentsFlowStarted())
```

Comment:
nit: could this call the property?

```suggestion
        analytics.track(event: WooAnalyticsEvent.SimplePayments.simplePaymentsFlowStarted())
```